src/readme/rules.ts

```diff
@@ -360,12 +360,26 @@ const rules: IRule[] = [
   },
   {
     available: true,
+    eslintRule: 'no-unsafe-finally',
+    tslintRule: 'no-unsafe-finally',
+    category: 'Possible Errors',
+    description: 'disallow control flow statements in finally blocks (recommended)',
+    eslintUrl: 'http://eslint.org/docs/rules/no-unsafe-finally',
+    tslintUrl: 'https://palantir.github.io/tslint/rules/no-unsafe-finally',
+    provider: 'native',
+    usage: `~~~json
+    "use-isnan": true
+    ~~~`
+  },
+  {
+    available: true,
     eslintRule: 'use-isnan',
     tslintRule: 'use-isnan',
     category: 'Possible Errors',
     description: 'disallow comparisons with the value `NaN` (recommended)',
     eslintUrl: 'http://eslint.org/docs/rules/use-isnan',
-    provider: 'tslint-eslint-rules',
+    tslintUrl: 'https://palantir.github.io/tslint/rules/use-isnan',
+    provider: 'native',
```

Comment:
@buzinas Since tslint made `use-isnan` we won't be able to use the one you made. I guess the change @jlowcs is actually good in retrospect. I think this change will have to go in but we will have two options:

- Delete the `use-is-nan` rule from this project
- rename it to `ter-use-is-nan` to have an alternative to the rule.

If we go for the second option I suggest we bump the major version and rename our rules to have the prefix so that no don't have conflicts with tslint in the future.